### PR TITLE
ci(github): add me and simon as codeowners

### PR DESCRIPTION
**Pull request by sndrs**

### CODE REVIEW

**Review by SiAdcock — APPROVED** [review]
The water's fiiiiiiiine 🛀

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @SiAdcock @sndrs
```
Comment (gtrufitt) [inline .github/CODEOWNERS:1]: 🥺 

```suggestion
* @SiAdcock @sndrs @gtrufitt
```

Comment (sndrs) [reply .github/CODEOWNERS]: of course!

Comment (SiAdcock) [reply .github/CODEOWNERS]: Just like old times

**Review by gtrufitt — APPROVED** [review]

**Pull request merged**